tn_api_hotel.yml

```diff
@@ -1080,3 +1121,130 @@ components:
                 description: The full text of the ruledescription
                 example: 'non-refundable rate cancellation permitted up to 1days before arrival'
 
+    HotelBookRequest:
+      required:
+        - super_trip_id
+        - passengers
+        - payment_info
```

Comment:
This isn't required is it? We can book one and have the payment be through 'cash' right? Although there's the guarantee bit sometimes - but not always.

Reply:
We talked about this